codegen/lib/graphql_java_gen.rb

```diff
@@ -274,4 +274,24 @@ def abstract_types
       end
     end
   end
+
+  def java_doc(element)
+    doc = ''
+    unless element.description.nil? || element.description == 0
+      description = element.description.gsub("\n", '')
```

Comment:
We should probably replace newlines with a space so that we don't end up squishing words together.